api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -12,31 +12,27 @@
 import static org.hamcrest.CoreMatchers.hasItem;
 import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.CoreMatchers.nullValue;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.mock;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Locale;
-import java.util.Set;
+import java.util.*;
```

Comment:
@PKatGITHUB can you please configure your IDE not to use star imports
see https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE